internal/provider/resources/resource_api_token_test.go

```diff
@@ -35,120 +35,10 @@ func TestAcc_ResourceOrganizationApiToken(t *testing.T) {
 		ProtoV6ProviderFactories: astronomerprovider.TestAccProtoV6ProviderFactories,
 		PreCheck:                 func() { astronomerprovider.TestAccPreCheck(t) },
 		CheckDestroy: resource.ComposeTestCheckFunc(
-			// Check that the organization api token has been removed
+			// Check that the api token has been removed
 			testAccCheckApiTokenExistence(t, checkApiTokensExistenceInput{name: apiTokenName, organization: true, shouldExist: false}),
 		),
 		Steps: []resource.TestStep{
-			// Test invalid role for token type
```

Comment:
Had to remove because now running into
```
    resource_api_token_test.go:34: Error running post-test destroy, there may be dangling resources: Check 1/1 error: api token should not exist
```

Reply:
whats the reason why all of those tests are failing? I see that there's this error but what is the root cause?

Reply:
actually it might just be one of these tests that are breaking it, will try to figure out which one

Reply:
last update: moved the validation to create and update functions instead of `ValidateConfig` and it fixed it.